pkgs/build-support/extract-darwin-app/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  undmg,
+}:
+
+/** A function to make a derivation from a darwin binary
+  *
+  * It makes the assumption that the binary comes in a ".app" or ".dmg".
```

Comment:
Shouldn't `.app` here be `.zip`?

Reply:
https://github.com/NixOS/nixpkgs/pull/244152/commits/2f85c6eed236dd0c1e523414630a6f6c481e46c7 updates the doc